CHANGELOG

```diff
@@ -10,6 +10,7 @@ v3.17 (Month 2020)
   * Remove tag color from issue titles in issue summary
   * Use shared noscript partial
   * Upgraded gems: puma, sass-rails
+  * Display note and evidence title on breadcrumb
```

Comment:
```suggestion
  * Display note and evidence titles in breadcrumbs
```

Reply:
please also move this entry up in the changelog so it is in alpha order.

Reply:
Thanks for the suggestion @MattBudz. Will update it.